### PR TITLE
Clarification of Customer Comment # 20231012

### PR DESCRIPTION
**Pull request by leanneeliatra**
### Description
A customer submitted a query in relation to this page of the documentation: https://opensearch.org/docs/latest/security/multi-tenancy/tenant-index/

**Customer query:** *The global tenant is not a primary tenant -> are you meaning your private tenant? -x*

**Answer to customers query:**
The term 'primary tenant' is not commonly used used, being more clear with the language would address this issue

**Proposed change to documentation:**
To change the documenation to use clearer languge, removing the term `primary tenant` in place for `global tenant` and `private tenant`, the terms that should be used in this context.

**Fix**: Update language in docs

### Issues Resolved
This ticket addresses **one part** of https://github.com/opensearch-project/documentation-website/issues/4314

### Checklist
- [x] By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license and subject to the [Developers Certificate of Origin](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

**Comment by hdhalter** [comment]
@scrawfor99 - Can you please review this? Thanks!

File: _security/multi-tenancy/tenant-index.md [hunk 1]
```diff
@@ -16,7 +16,7 @@ redirect_from:
 - **Private** -- This tenant is exclusive to each user and can't be shared. It does not allow you to access routes or index patterns created by the user's global tenant.
 - **Custom** -- Administrators can create custom tenants and assign them to specific roles. Once created, these tenants can then provide spaces for specific groups of users.
 
-The global tenant is not a *primary* tenant in the sense that it replicates its content in a private tenant. To the contrary, if you make a change to your global tenant, you won't see that change reflected in your private tenant. Some example changes include the following:
+The global tenant in OpenSearch Dashboards doesn't synchronize its content with private tenants. When you make modifications within your global tenant, these changes are exclusive to the global tenant. They aren't automatically mirrored or replicated in the private tenant. Some example changes include the following:
```
Comment (Naarcha-AWS) [inline _security/multi-tenancy/tenant-index.md:19]:
```suggestion
The global tenant in OpenSearch Dashboards doesn't synchronize its content with private tenants. When you make modifications within your global tenant, these changes are exclusive to the global tenant. They aren't automatically mirrored or replicated in the private tenant. Some example changes that are exclusive to the global tenant include the following:
```

**Review by Naarcha-AWS — CHANGES_REQUESTED** [review]

**Review by scrawfor99 — APPROVED** [review]
Signing off for technical accuracy. Looks good!

File: _security/multi-tenancy/tenant-index.md (hunk 1 above)
Comment (leanneeliatra) [reply _security/multi-tenancy/tenant-index.md:19]: Hi @Naarcha-AWS, the operations listed:
Create visualizations
Create index patterns
Are not exclusive to only the global tenant so I think the new wording will be misleading.

Comment (Naarcha-AWS) [reply _security/multi-tenancy/tenant-index.md:19]: @leanneeliatra: I removed that part of the sentence. Let me know if this looks okay.

Comment (leanneeliatra) [reply _security/multi-tenancy/tenant-index.md]: I'm not sure how to 'mark the requested changes are addressed', but once that is complete, this all looks good to be merged now. Thanks!

**Review by Naarcha-AWS — APPROVED** [review]

File: _security/multi-tenancy/tenant-index.md [hunk 2]
```diff
@@ -16,7 +16,7 @@ redirect_from:
 - **Private** -- This tenant is exclusive to each user and can't be shared. It does not allow you to access routes or index patterns created by the user's global tenant.
 - **Custom** -- Administrators can create custom tenants and assign them to specific roles. Once created, these tenants can then provide spaces for specific groups of users.
 
-The global tenant is not a *primary* tenant in the sense that it replicates its content in a private tenant. To the contrary, if you make a change to your global tenant, you won't see that change reflected in your private tenant. Some example changes include the following:
+The global tenant in OpenSearch Dashboards doesn't synchronize its content with private tenants. When you make modifications within your global tenant, these changes are exclusive to the global tenant. They aren't automatically mirrored or replicated in the private tenant. Some example changes including the following:
```
Comment (Naarcha-AWS) [inline _security/multi-tenancy/tenant-index.md:19]:
```suggestion
The global tenant in OpenSearch Dashboards doesn't synchronize its content with private tenants. When you make modifications within your global tenant, these changes are exclusive to the global tenant. They aren't automatically mirrored or replicated in the private tenant. Some example changes to both private and global tenants include the following:
```

File: _security/multi-tenancy/tenant-index.md [hunk 3]
```diff
@@ -16,7 +16,7 @@ redirect_from:
 - **Private** -- This tenant is exclusive to each user and can't be shared. It does not allow you to access routes or index patterns created by the user's global tenant.
 - **Custom** -- Administrators can create custom tenants and assign them to specific roles. Once created, these tenants can then provide spaces for specific groups of users.
 
-The global tenant is not a *primary* tenant in the sense that it replicates its content in a private tenant. To the contrary, if you make a change to your global tenant, you won't see that change reflected in your private tenant. Some example changes include the following:
+The global tenant in OpenSearch Dashboards doesn't synchronize its content with private tenants. When you make modifications within your global tenant, these changes are exclusive to the global tenant. They aren't automatically mirrored or replicated in the private tenant. Some example changes to both private and global tenants include the following:
```
Comment (Naarcha-AWS) [inline _security/multi-tenancy/tenant-index.md:19]:
```suggestion
The global tenant in OpenSearch Dashboards doesn't synchronize its content with private tenants. When you make modifications inside your global tenant, these changes are exclusive to the global tenant. They aren't automatically mirrored or replicated in the private tenant. Some example changes to both private and global tenants include the following:
```

**Review by Naarcha-AWS — APPROVED** [review]

**Pull request merged**